contents/blog/posthog-vs-growthbook.md

```diff
@@ -0,0 +1,637 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Multiple customizable insights for a flag</td>
+        </tr>
+       <tr>
+            <td><strong>Non-flag data</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Combine flag data with other product data</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+## Pricing
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <th></th>
+            <th className="text-center">PostHog</th>
+            <th className="text-center">GrowthBook</th>
+        </tr>
+    </thead>
+    <tbody>
+       <tr>
+            <td><strong>Pricing</strong></td>
+            <td className="text-center">Per request</td>
+            <td className="text-center">Seat-based</td>
+        </tr>
+       <tr>
+            <td><strong>Free plan</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+       <tr>
+            <td><strong>Free experiments</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+        </tr>
+       <tr>
+            <td><strong>Free collaboration</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center">3 seats free, then $20/seat/month</td>
+        </tr>
+       <tr>
+            <td><strong>Fully transparent</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog’s [feature flag pricing](/pricing?product=feature-flags) is pay-per-request (and A/B tests use feature flags). There is a generous free tier of 1M requests per month with all features, add-ons, and integrations available.
+
+![Pricing](../images/blog/posthog-vs-growthbook/pricing.png)
+
+Like PostHog, GrowthBook is free to self-host. GrowthBook Cloud is a seat-based model. It is free for up to 3 users, then $20/user/month. They claim this provides "unlimited traffic," but in reality, the limit is 10M requests per month, after which you need an enterprise plan.
+
+![Growthbook](../images/blog/posthog-vs-growthbook/growthbook.png)
+
+Features, like flag scheduling, permissions, custom fields, and the visual editor are only available on the GrowthBook Pro paid plan. 
+
+#### Example scenarios
```

Comment:
```suggestion
#### Example scenarios
```

Chef kiss. 

It may be worth changing the header to something more SEOable: "How much does GrowthBook cost?" or "Which costs more, PostHog or GrowthBook?" etc, but I defer to you.